bazel/osx/crosstool/CROSSTOOL.tpl

```diff
@@ -1270,8 +1275,7 @@ toolchain {
     config_name: "c++-link-interface-dynamic-library"
     action_name: "c++-link-interface-dynamic-library"
     tool {
-      tool_path: "wrapped_clang"
-      execution_requirement: "requires-darwin"
+      tool_path: "DUMMY_TOOL"
```

Comment:
I wanted to submit a simple project as a test case for the ar bug, but after I was unable to reproduce it, I realized that the [0.5.2](https://github.com/bazelbuild/bazel/tree/0.5.2) tag is not bazelbuild/bazel@7b85122c9ed1c5bc32ef9c6ac213910adee735e2, the latter being the SHA from which other stuff was copied.

I think what's happened is that a 0.5.2 release branch was started, and the ar bug was fixed there. Meanwhile in master the DUMMY_TOOL entries were removed (presumably because they weren't being used) and the ar bug was also fixed there, but after 7b85122c. I'm at a loss to explain what actually fixes the bug or why the DUMMY_TOOL entries don't seem to bother anything.